File: Scripts/app/model/CrateDescriptionDTO.ts
module dockyard.model {
    export class CrateDescriptionDTO {
		manifestId: number;
        manifestType: string;
        availability: AvailabilityType;
        sourceActivityId: string;
        label: string;
        producedBy: string;
        selected: boolean;
        fields: FieldDTO[];
	}
}

File: Data/Interfaces/DataTransferObjects/CrateDescriptionDTO.cs
using Newtonsoft.Json;

namespace Data.Interfaces.DataTransferObjects
{
    public class CrateDescriptionDTO
    {
        [JsonProperty("manifestId")]
        public int ManifestId { get; set; }

        [JsonProperty("manifestType")]
        public string ManifestType{ get; set; }

        [JsonProperty("label")]
        public string Label{ get; set; }

        [JsonProperty("producedBy")]
        public string ProducedBy { get; set; }

        [JsonProperty("selected")]
        public bool Selected { get; set; }

        public CrateDescriptionDTO()
        {
            Selected = false;
        }
    }
}
